Source/WebKit/NetworkProcess/ServiceWorker/WebSWServerToContextConnection.cpp

```diff
@@ -340,11 +345,11 @@ void WebSWServerToContextConnection::startFetch(ServiceWorkerFetchTask& task)
 
 void WebSWServerToContextConnection::didReceiveFetchTaskMessage(IPC::Connection& connection, IPC::Decoder& decoder)
 {
-    auto iterator = m_ongoingFetches.find(ObjectIdentifier<FetchIdentifierType>(decoder.destinationID()));
-    if (iterator == m_ongoingFetches.end())
+    auto fetch = m_ongoingFetches.get(ObjectIdentifier<FetchIdentifierType>(decoder.destinationID()));
+    if (!fetch)
         return;
 
-    iterator->value->didReceiveMessage(connection, decoder);
+    Ref { *fetch }->didReceiveMessage(connection, decoder);
```

Comment:
... and just use `fetch->` here.

Reply:
So that's not enough because iterator->value is a value in heap, not in stack. We need to store it in a local variable.